CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## [UNRELEASED] · YYYY-MM-DD
 ### 🌀 Changed
 - Reformat sources with [Blue](https://github.com/JuliaDiff/BlueStyle) ([#12](https://github.com/cbrnr/XDF.jl/pull/12) by [Alberto Barradas](https://github.com/abcsds))
+### ✨ Added
+- Add tests for string markers from issue xdf-modules/libxdf#19 (([#13](https://github.com/cbrnr/XDF.jl/pull/13) by [Alberto Barradas](https://github.com/abcsds)))
```

Comment:
I think we should be more specific here. You fixed an issue with string streams consisting of multiple channels, right? It would be more helpful to explicitly mention this here instead of linking to an issue. Also, I think this change should then go into the "Fixed" section, WDYT?

Reply:
What did you mention is the template to follow? I can't find the emoji for the fixed section. I thought you mentioned a repo from xdf-modules, but their changelogs don't have emojis. I can't find the pr where it was mentioned. I thought it was #11.